src/Microsoft.PowerShell.ScheduledJob/ScheduledJobDefinition.cs

```diff
@@ -2385,42 +2385,6 @@ public ScheduledJobInvocationInfo(JobDefinition definition, Dictionary<string, o
 
         #endregion
 
-        #region ISerializable Implementation
```

Comment:
`ScheduledJob` module is excluded from our build (we don't ship the `ScheduledJob` module), so we should not make changes to its code.

Reply:
excluded these changes